backend/routes/content.py

```
@@ -0,0 +1,261 @@
+"""
+Content management API routes.
+
+Provides endpoints for:
+- Content gap detection and management
+- Popular topic analytics
+- Source usage analytics
+"""
+
+import logging
+import sqlite3
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)